pallets/runtime/common/src/lib.rs

```diff
@@ -45,8 +45,8 @@ pub type NegativeImbalance<T> =
 
 parameter_types! {
     pub const BlockHashCount: BlockNumber = 250;
-    /// We allow for 2 seconds of compute with a 6 second average block time.
-    pub const MaximumBlockWeight: Weight = 2 * WEIGHT_PER_SECOND;
+    /// We allow for 0.5 seconds of compute with a 6 second average block time.
+    pub const MaximumBlockWeight: Weight = (5 * WEIGHT_PER_SECOND) / 10;
```

Comment:
Minor: `(5 * WEIGHT_PER_SECOND) / 10` can be compressed to ` WEIGHT_PER_SECOND / 2`. Alternatively, we can also use `500 * WEIGHT_PER_MILLISECOND`.